std/conv.d

```diff
@@ -1939,7 +1939,7 @@ Target parse(Target, Source)(ref Source s)
     }
     else
     {
-        // Larger than int types
+        // int or larger types
```

Comment:
This was a very misleading comment. Made me miss the code below and wonder where the int parsing code went.